be/src/storage/lake/transactions.cpp

```diff
@@ -36,6 +36,13 @@ StatusOr<TabletMetadataPtr> publish_version(TabletManager* tablet_mgr, int64_t t
     VLOG(1) << "publish version tablet_id: " << tablet_id << ", txns: " << JoinInts(txn_ids, ",")
             << ", base_version: " << base_version << ", new_version: " << new_version;
 
+    auto new_metadata_path = tablet_mgr->tablet_metadata_location(tablet_id, new_version);
+    auto cached_new_metadata = tablet_mgr->metacache()->lookup_tablet_metadata(new_metadata_path);
+    if (cached_new_metadata != nullptr) {
+        LOG(INFO) << "Found cached tablet metadata for version " << new_version << " of tablet " << tablet_id;
```

Comment:
this logline may be too chatty?

also this line doesn't tell what it means when the new version of meta is found from cache when doing publish

Reply:
Updated